mmdeploy/pytorch/ops/pad.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.onnx.symbolic_helper as sym_help
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
```

Comment:
Is this modified from the PyTorch source code?  If it is, we better add a comment here.